src/sql/src/statements.rs

```diff
@@ -84,7 +84,7 @@ fn parse_string_to_value(
             }
         }
         ConcreteDataType::DateTime(_) => {
-            if let Ok(datetime) = common_time::datetime::DateTime::from_str(&s) {
+            if let Ok(datetime) = common_time::datetime::DateTime::from_str_local(&s) {
```

Comment:
Why one is `from_str_utc` and another is `from_str_local`?

Reply:
Because the old DateTime `from_str` implementation uses the local timezone when the string doesn't contain any timezone info. I keep it the same way.


https://github.com/GreptimeTeam/greptimedb/blob/966875ee1120bc0189d6303d4ea8fb445cdcf7a7/src/common/time/src/datetime.rs#L69

Reply:
>when the timezone is not provided both in string and parameter.

We should then fallback to server timezone, then UTC+0. Local timezone isn't one of the options

Reply:
Local timezone is the system timezone.